components/script/dom/htmllinkelement.rs

```diff
@@ -235,6 +237,14 @@ impl HTMLLinkElement {
             }
         };
 
+        // Step 3
+        let cors_setting = match self.GetCrossOrigin() {
```

Comment:
I have added a method in `request.rs` which given a `&str` returns `CorsSettings`. I am not sure if this is good refactoring. Other option could be return `CorsSettings` given `DomString` but I dont know what is the right location for such utility.